### PR TITLE
Elnaqeeb lcd testing

### PR DESCRIPTION
**Pull request by ahmedashraf96**

### CODE REVIEW

File: Elnaqeeb/LCD_Testing/LCD_TESTING.c [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        printf("Test case 1 : FAILED!\n");
+    }
+
+    /*
+     * Test case 2:
+     *  Sending command to the LCD.
+     *  SUCCESS: Returns OPERATION_SUCCESS
+     *  FAILED : Other  
+     */    
+    ret_err = LCD_sendCommand(LCD_CMD);
```
Comment (YehiaSprints) [inline Elnaqeeb/LCD_Testing/LCD_TESTING.c]: What if the command is wrong?

File: Elnaqeeb/LCD_Testing/LCD_TESTING.c [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        printf("Test case 2 : FAILED!\n");
+    }
+
+    /*
+     * Test case 3:
+     *  Displaying certain character over the LCD.
+     *  SUCCESS: Returns OPERATION_SUCCESS
+     *  FAILED : Other  
+     */    
+    ret_err = LCD_displayChar('A');
```
Comment (YehiaSprints) [inline Elnaqeeb/LCD_Testing/LCD_TESTING.c]: What if the character cannot be displayed on the LCD, e.g: Arabic characters?

Comment (YehiaSprints) [reply Elnaqeeb/LCD_Testing/LCD_TESTING.c]: What if the LCD is not initialized?

File: Elnaqeeb/LCD_Testing/LCD_TESTING.c [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+        printf("Test case 5 : FAILED!\n");
+    }
+
+    /*
+     * Test case 6:
+     *  Passing certain row and column so that LCD can begin writing data from them.
+     *  SUCCESS: Returns OPERATION_SUCCESS
+     *  FAILED : Other  
+     */    
+    ret_err = LCD_gotoRowColumn(ROW_0 , COL_0);
```
Comment (YehiaSprints) [inline Elnaqeeb/LCD_Testing/LCD_TESTING.c]: What if the row or column is wrong?

**Comment by YehiaSprints** [comment]
Dear Naqeeb,

Here is Mohammed Tarek's comment:
The initialization was not done through the OS, It's not recommended to use interrupts aside from the OS because that makes your system unpredictable.

**Comment by BassamSharkawy-Sprints** [comment]
Please apply Yahia's/M.Tarek's reviews and merge.

**Pull request merged**